content/tutorials/manage-the-rippled-server/troubleshooting/server-doesnt-sync.md

```diff
@@ -0,0 +1,110 @@
+# rippled Server Doesn't Sync
+
+This page explains possible reasons [a `rippled` server](the-rippled-server.html) may start successfully, but get stuck in a ["connected" state](rippled-server-states.html) without ever fully connecting to the network. (If the server crashes during or shortly after startup, see [Server Won't Start](server-wont-start.html) instead.)
+
+These instructions assume you have [installed `rippled`](install-rippled.html) on a supported platform.
+
+
+## Normal Syncing Behavior
+
+Syncing with the network normally takes about 5 to 15 minutes. During that time, the server does several things:
+
+- Loads a recommended validator list (typically from `vl.ripple.com`) to determine which validators it trusts.
+- [Discovers peer servers](peer-protocol.html#peer-discovery) and connects to them.
+- Downloads the [header](ledger-header.html) and full [state information](ledgers.html#tree-format) of the latest ledger from its peers, and uses that to build its internal database of ledger data.
+- Listens to its trusted validators to find which ledger hashes have been recently validated.
+- Collects newly-broadcast transactions and attempts to apply them to its in-progress ledger.
+
+If the server is unable to keep up with the network while doing these tasks, the server does not sync to the network.
+
+
+## First Step: Restart
+
+Many syncing issues can be resolved by simply restarting the server. Even if it failed to sync for whatever reason, it may succeed on the second try.
```

Comment:
"Regardless of the reason it failed to sync on the first try, it may succeed on the second try."

Reply:
I'm going with the slightly plainer "No matter why it failed to sync the first time, it may succeed on the second try."